packages/google_maps_flutter/lib/src/bitmap.dart

```diff
@@ -58,8 +58,9 @@ class BitmapDescriptor {
     String assetName, {
     AssetBundle bundle,
     String package,
+    bool mipmaps = true,
```

Comment:
Is it possible to make it a non-breaking change by defaulting the `mipmaps` to false?

Reply:
Sorry, this is non-breaking already.  mipmaps = true is how it worked before

Reply:
Since the ecosystem might have already adopted the last change, compares to the latest code base, setting `mipmaps` to true would be a breaking change, right?